src/util/postgres_client.js

```diff
@@ -270,7 +271,39 @@ class PostgresTable {
             dbg.error('got error on _init_table:', err);
             throw err;
         }
-        // TODO: create indexes
+
+        if (this.db_indexes) {
+            try {
+                await Promise.all(this.db_indexes.map(async index => {
+                    const { fields, options = {} } = index;
+                    try {
+                        const index_name = options.name || Object.keys(fields).join('_');
+                        dbg.log0(`creating index ${index_name} in table ${this.name}`);
+                        const col_arr = [];
+                        _.forIn(fields, (value, key) => {
+                            col_arr.push(`(data->>'${key}') ${value > 0 ? 'ASC' : 'DESC'}`);
+                        });
+                        const col_idx = `(${col_arr.join(',')})`;
+                        const uniq = options.unique ? 'UNIQUE' : '';
+                        const partial = options.partialFilterExpression ? `WHERE ${mongo_to_pg('data', options.partialFilterExpression)}` : '';
+                        const idx_str = `CREATE ${uniq} INDEX idx_btree_${this.name}_${index_name} ON ${this.name} USING BTREE ${col_idx} ${partial}`;
```

Comment:
shouldn't we check for index existence before creating?
maybe this is helpful (didn't check it myself) https://stackoverflow.com/questions/45983169/checking-for-existence-of-index-in-postgresql/45983490